kucher/view/utils.py

```diff
@@ -78,7 +78,7 @@ def get_monospace_font(small=False) -> QFont:
 @cached
 def _get_monospace_font_impl(small=False) -> QFont:
     begun = time.monotonic()
-    multiplier = 0.8 if small else 1.0
+    multiplier = 0.9 if small else 1.0
```

Comment:
What I was referring to in the issue description is that while the font was too small on Windows, it was of just the right size on GNU/Linux systems. So I am thinking that we should constrain the absolute size of the selected font instead of changing the multiplier. Perhaps like `max(12, QFont().pointSize() * multiplier)`, where 12 should be defined as a constant somewhere in this module, and its value should be determined empirically.